src/signals/incident/containers/IncidentContainer/saga.js

```diff
@@ -39,7 +33,7 @@ export function* getClassification(action) {
       `${configuration.CATEGORIES_ENDPOINT}${category}/sub_categories/${subcategory}`
     );
 
-    yield put(getClassificationSuccess(getClassificationData(classification)));
+    yield put(getClassificationSuccess(getClassificationData(category, subcategory, classification)));
```

Comment:
Why is it necessary to call `getClassificationData` here when the payload of the action already contains the return value of `getClassificationData`? As far as I can see, only the value of the `classification` prop from the   `getClassificationData` result is used by the reducer.

Reply:
All the fields returned are used in the reducer. the results of the `getIncidentClassification` will update all the fields from `getClassificationData` in the `incident`.

Reply:
Not true, only `classification` is used by `action.type` `GET_CLASSIFICATION_SUCCESS`